### DISCUSSION TITLE
%superior_island_exists% output

### DISCUSSION THREAD

**Discussion by Davisiiiik** [BG-Software-LLC/SuperiorSkyblock2#1101] [Implemented Feature Requests] [answered]
Okay, so someone changed output of the %superior_island_exists% placeholder to ✔/no from yes/no. Can you please change it back? It broke all my custom commands relying on this placeholder (and comparing colored symbol is really not ideal). You can format any text using other placeholders, so I think this isnt good change at all.

**Comment by Davisiiiik** [comment, marked as answer]
Answered on Discord

**Comment by OmerBenGera** [comment]
Just so others can know, you can change the output of the placeholder in the lang files. 
I also reverted it so by default it will show "Yes" and "No" again.
